scripts/benchmark.sh

```diff
@@ -0,0 +1,244 @@
+#!/bin/bash
+WORKDIR=$(pwd)
+OUTPUT_DIR="${WORKDIR}/out"
+REPEAT=1
+SERVERS=("172.20.0.2" "172.20.0.3" "172.20.0.4" "172.20.0.5")
+CLUSTER_PEERS=(
+    ""
+    "${SERVERS[2]}:2379 ${SERVERS[3]}:2379"
+    "${SERVERS[1]}:2379 ${SERVERS[3]}:2379"
+    "${SERVERS[1]}:2379 ${SERVERS[2]}:2379"
+)
+XLINE_TEST_CASE=(
```

Comment:
Can we make the shell simpler as there're many duplications.

Reply:
Yes, but cannot quickly run the specified test_case